src/ServerMonitor.js

```diff
@@ -164,10 +164,12 @@ export default class ServerMonitor {
             newStatusMessageBuilder.push(staffMemberMessage);
         }
 
+        newStatusMessageBuilder.push(`\nNext update in ${this.timestamp(Date.now() + config.check_interval)}`);
```

Comment:
The timestamp needs to be divided by 1000 since Discord uses seconds instead of milliseconds and then floored to make it an integer